README.rst

```diff
@@ -44,6 +44,33 @@ Features
 - Convenience methods are provided for commonly used functionality.
 - Exceptions are raised when errors occur.
 
+Example
+-------
+
+.. code-block:: python
+
+    from imapclient import IMAPClient
+
+    # context manager ensures the session is cleaned up
+    with imapclient.IMAPClient(host="imap.host.org") as client:
```

Comment:
`imapclient` is not imported. It should be just `IMAPClient`.

Reply:
Good catch. Fixed.